src/pkg/cli/compose/context_test.go

```diff
@@ -75,7 +75,6 @@ func TestUploadArchive(t *testing.T) {
 				t.Errorf("Expected %v, got %v", server.URL+path, url)
 			}
 		})
-
```

Comment:
@KevyVo your change had go linter errors: please add the GIT hooks so you run it before push.